zzk/service/hoststate.go

```diff
@@ -157,25 +157,16 @@ func (l *HostStateListener) Spawn(shutdown <-chan interface{}, stateID string) {
 
 		// load the service state node
 		sspth := path.Join("/services", serviceID, stateID)
-		ssdat := &ServiceState{}
-		if err := l.conn.Get(sspth, ssdat); err == client.ErrNoNode {
-
+		exists, err := l.conn.Exists(sspth)
```

Comment:
I tried that, and it broke the code. Glenn and I backed this out and restored the Get. I'll be happy to revisit the change, but I'll need your help.